utils/python/CIME/build.py

```diff
@@ -79,9 +80,12 @@ def build_model(case, build_threaded, exeroot, clm_config_opts, incroot, complis
         time.sleep(1)
 
     # aquap has a dependancy on atm so we build it after the threaded loop
-    if "aquap" in complist:
-        _build_model_thread(config_ocn_dir, caseroot, bldroot, "aquap", file_build,
-                            exeroot, "ocn", "aquap", objdir, incroot, thread_bad_results)
+
+    for model, comp, nthrds, ninst, config_dir in complist:
```

Comment:
I think this is fine, other option:

    if "aquap" in [item[1] for item in complist]: